components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -558,9 +554,11 @@ subroutine radiation_init(state,pbuf)
       !needed for SPA
       integer :: year, month, day, tod, next_month
       real(r8), pointer :: aerosol_optical_property(:,:,:,:)
-      real(r8), pointer :: aerosol_optical_property_lw(:,:,:,:)
-      real(r8), pointer :: aerosol_optical_property_sw(:,:,:,:)
+      real(r8), pointer :: aerosol_optical_property_lw(:,:,:,:,:)
+      real(r8), pointer :: aerosol_optical_property_sw(:,:,:,:,:)
 
+      nullify(aerosol_optical_property_lw)
+      nullify(aerosol_optical_property_sw)
       !-----------------------------------------------------------------------
```

Comment:
Why would you need to nullify a pointer immediately after creating it? How could it possibly be assigned at this point?